cdk/lib/support-reminders.ts

```diff
@@ -1,15 +1,264 @@
+			securityGroups,
+			environment,
+			handler: "create-reminder-signup/lambda/lambda.handler",
+			functionName: `support-reminders-create-reminder-signup-${this.stage}`,
+		});
+
+		const reactivateRecurringReminderLambda = new GuLambdaFunction(this, "reactivate-recurring-reminder", {
+			app,
+			runtime,
+			fileName,
+			vpc,
+			vpcSubnets,
+			securityGroups,
+			environment,
+			handler: "reactivate-recurring-reminder/lambda/lambda.handler",
+			functionName: `support-reminders-reactivate-recurring-reminder-${this.stage}`,
+		});
+
+		const cancelRemindersLambda = new GuLambdaFunction(this, "cancel-reminders", {
+			app,
+			runtime,
+			fileName,
+			vpc,
+			vpcSubnets,
+			securityGroups,
+			environment,
+			handler: "cancel-reminders/lambda/lambda.handler",
+			functionName: `support-reminders-cancel-reminders-${this.stage}`,
+		});
+
+
+		// ---- API gateway ---- //
+		const supportRemindersApi = new GuApiGatewayWithLambdaByPath(this, {
+			app,
+			defaultCorsPreflightOptions: {
+				allowOrigins: Cors.ALL_ORIGINS,
+				allowMethods: Cors.ALL_METHODS,
+				allowHeaders: ["Content-Type"],
+			},
+			monitoringConfiguration: {
+				snsTopicName: "conversion-dev",
+				http5xxAlarm: {
+					tolerated5xxPercentage: 1,
+				}
+			},
+			targets: [
+				{
+					path: "/reactivate",
+					httpMethod: "POST",
+					lambda: reactivateRecurringReminderLambda,
+				},
+				{
+					path: "/create/recurring",
+					httpMethod: "POST",
+					lambda: createRemindersSignupLambda,
+				},
+				{
+					path: "/search",
+					httpMethod: "POST",
+					lambda: searchRemindersLambda,
+				},
+				{
+					path: "/create/one-off",
+					httpMethod: "POST",
+					lambda: createRemindersSignupLambda,
+				},
+				{
+					path: "/cancel",
+					httpMethod: "POST",
+					lambda: cancelRemindersLambda,
+				},
+			],
+		})
+
+
+		// ---- Scheduled lambda functions ---- //
+		new GuScheduledLambda(this, "signup-exports", {
+			app,
+			runtime,
+			fileName,
+			vpc,
+			vpcSubnets,
+			securityGroups,
+			environment,
+			handler: "signup-exports/lambda/lambda.handler",
+			functionName: `support-reminders-signup-exports-${this.stage}`,
+			rules: [
+				{
+					schedule: Schedule.expression("cron(05 00 * * ? *)"),
+				},
+			],
+			monitoringConfiguration: {
+				snsTopicName: "conversion-dev",
+				toleratedErrorPercentage: 1,
+			},
+		});
+
+		new GuScheduledLambda(this, "next-reminders", {
+			app,
+			runtime,
+			fileName,
+			vpc,
+			vpcSubnets,
+			securityGroups,
+			environment,
+			handler: "next-reminders/lambda/lambda.handler",
+			functionName: `support-reminders-next-reminders-${this.stage}`,
+			rules: [
+				{
+					schedule: Schedule.expression("cron(05 00 * * ? *)"),
+				},
+			],
+			monitoringConfiguration: {
+				snsTopicName: "conversion-dev",
+				toleratedErrorPercentage: 1,
+			},
+		});
+
+
+		// ---- DNS ---- //
+		const certificateArn = `arn:aws:acm:eu-west-1:${this.account}:certificate/${props.certificateId}`;
```

Comment:
We should create this certificate in this stack too, though I appreciate this differs from the template being migrated. Let's update it in a follow-up PR.